### PR TITLE
add rustfmt.toml to configure format

### PR DESCRIPTION
**Pull request by Mossaka**
Closes https://github.com/deislabs/containerd-wasm-shims/issues/156

@radu-matei this overlaps with your PR #63

### CODE REVIEW

**Comment by Mossaka** [comment]
Could you please give it a review? @devigned @kate-goldenring

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -13,10 +13,11 @@ jobs:
         with:
           workspaces: |
             "containerd-shim-* -> target"
-      - name: "Install dependencies"
+      - run: 
+          rustup toolchain install nightly --component rustfmt
```
Comment (kate-goldenring) [inline .github/workflows/ci.yaml:17]: Why are we using nightly?

File: rustfmt.toml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+newline_style = "Native"
+unstable_features = true # Cargo fmt now needs to be called with `cargo +nightly fmt`
```
Comment (devigned) [inline rustfmt.toml:2]: @kate-goldenring I think this is the reason for nightly

**Review by devigned — APPROVED** [review]
lgtm

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (Mossaka) [reply .github/workflows/ci.yaml:17]: rustfmt.toml requires unstable features

**Pull request merged**